packages/react-liveness/src/components/FaceLivenessDetector/shared/FaceLivenessErrorModal.tsx

```diff
@@ -67,9 +67,17 @@ const renderToastErrorModal = (props: {
 
   return (
     <>
-      <Flex className={LivenessClassNames.ErrorModal}>
+      <Flex
+        className={LivenessClassNames.ErrorModal}
+        id="amplify-liveness-error-description"
```

Comment:
It feels off to me that a "description" element is wrapping a "heading" element?